cypress/support/index.js

```diff
@@ -27,6 +29,15 @@ Cypress.on('uncaught:exception', (err) => {
   }
 });
 
-if (Cypress.env('SECURITY_ENABLED')) {
-  Cypress.env('openSearchUrl', 'https://localhost:9200');
+// Proxy layer of AWS OpenSearch domain redirects to login page
+//  if you haven't authenticate
+if (Cypress.env('AWS_DOMAIN')) {
```

Comment:
What about an opensource domain with security enabled? I don't think we should remove the existing logic.

Reply:
`SECURITY_ENABLED` is not supposed to tuning the url, because we provide url directly through `openSearchUrl`